Add tracking endpoint

webmasterjunkie opened the pull request:
Also changes the campaign list endpoint to include the optional status
parameter.

webmasterjunkie committed:
Add tracking endpoint

Also changes the campaign list endpoint to include the optional status
parameter.

pete001 commented on src/apiary.apib:
```diff
@@ -879,10 +879,14 @@ Before a Publishers can begin to generate conversions, they must be associated w
 + Parameters
     + publisher_id: 1l1007802 - ID of the publisher
+    + status: (string, optional) - The status of the publisher on the campaign
+        + a - approved
```
This should be a data structure, enum.

pete001 commented on src/apiary.apib:
```diff
@@ -892,6 +896,23 @@ View all Campaigns that the Publisher is associated with, along with the status
++ Parameters
+    + publisher_id: 1l1007802 - ID of the publisher
+    + status: (string) - The status of the publisher on the campaign
+        + a - approved
```
It looks like there the following in common.apib: 

```
## Status (enum)
+ a - Approved
+ p - Pending
+ r - Rejected
+ n
```

pete001 replied:
If `n` isnt valid, then we you could create a new status for a/p/r and `Include` that into this existing Status (enum)

webmasterjunkie committed:
Change data type

https://github.com/PerformanceHorizonGroup/apidocs/pull/14/files#r699794
68
https://github.com/PerformanceHorizonGroup/apidocs/pull/14/files#r699794
87

pete001 commented on src/apiary.apib:
```diff
@@ -892,6 +896,23 @@ View all Campaigns that the Publisher is associated with, along with the status
+
++ Parameters
+    + publisher_id: 1l1007802 - ID of the publisher
+    + status: (enum) - The status of the publisher on the campaign
```
This is required in this instance.

webmasterjunkie replied:
Correct. The docs show the absence of (required) defaults to [required](https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#description-13). I have added the data type for specifics now.

webmasterjunkie committed:
Add data types
